src/kernel/mempool_limits.h

```diff
@@ -24,6 +24,15 @@ struct MemPoolLimits {
     int64_t descendant_count{DEFAULT_DESCENDANT_LIMIT};
     //! The maximum allowed size in virtual bytes of an entry and its descendants within a package.
     int64_t descendant_size_vbytes{DEFAULT_DESCENDANT_SIZE_LIMIT_KVB * 1'000};
+
+    /**
+     * @return MemPoolLimits with all the limits set to the maximum
+     */
+    static MemPoolLimits NoLimits()
```

Comment:
67a4932a2ca1131ea04a011d67b3f3c2da727b5f
```suggestion
    static constexpr MemPoolLimits NoLimits()
```

Reply:
No strong view. It's currently not called at compile time so perhaps a bit premature, but simultaneously it seems unlikely we'll ever want to not guarantee this fn to be `constexpr` so might as well add it already. Updated.

Reply:
As @hebasto kindly explained offline (comparing https://godbolt.org/z/78r3sY4TW with https://godbolt.org/z/TfvKKvdM9), using `constexpr` here does indeed have benefits as the compiler is able to replace the `NoLimits()` function calls with constants. Thank you!